.github/workflows/pr.yaml

```diff
@@ -72,6 +72,23 @@ jobs:
       - name: Lint python
         run: yarn lint-python
 
+  lint-jinja:
+    runs-on: ubuntu-22.04
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install node dependencies
+        run: yarn install --immutable
+
+      - name: Install python dependencies
+        run: |
+          python3 -m pip install --upgrade pip
+          sudo pip3 install djlint
```

Comment:
If djlint is in `requirements.txt` as a dependency, does it need to be installed separately here?

Reply:
you're right, it doesn't need to be in `requirements.txt`, I will remove

Reply:
But if it's removed from `requirements.txt` it will not be installed in development environment, right? So devs will have to install it manually?

I believe it should stay in `requirements.txt`, but maybe in CI we can install dependencies from requirements.txt instead of manually installing djlint?

I'm not sure how it works with other Python based linting, like black.

Reply:
Hmm… ok it seems that we are installing black on CI directly, so I guess it's fine to do the same with djlint

https://github.com/canonical/canonical.com/blob/13a3265e6f140c8811eb4053a60006a7a9a3367b/.github/workflows/pr.yaml#L70

Reply:
It looks like black and flake8 are part of dotrun image, so that's why they are installed by default in dev environment.

Until djlint is added to dotrun, I think it needs to be in requirements.txt of a project, so it is installed in dev environment:

https://github.com/canonical/dotrun-image/blob/63768e54385e465c2a2e29155613d3d7e313eb06/src/setup.py#L13-L20

Reply:
ok leaving it there then